third_party/ti/CMakeLists.txt

```diff
@@ -38,4 +38,16 @@ if(OT_PLATFORM STREQUAL "cc2650")
         INTERFACE
             -T${CMAKE_CURRENT_SOURCE_DIR}/devices/cc26x0/linker_files/cc26x0f128.lds
     )
+elseif(OT_PLATFORM MATCHES "cc(13|26)[0-9]2")
```

Comment:
It may be fine for now, but we are not sure if there would be other platforms matching the regex but requiring other drivers.
How about explicitly matching the exact platforms and combining with `OR`?

Reply:
Ok, will compare the specific platforms name